content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -408,6 +408,35 @@ validation and auto-completion.
 ⚠️ Note that using the `checkpoint` field in `dvc.yaml` is not compatible with
 `dvc repro`.
 
+## Plots definitions
+
+The list of plots contains one or more user-defined
```

Comment:
@dberenbaum @pared @daavoo what is the usual case that is expected for this global "plots" declaration? Is it primarily about bootstrapping the repo initially that doesn't have any stages?) should we clarify this? make example simpler below?

from the example it's not clear what it the benefit of having it outside the pipeline ..

Reply:
I would say it's explained in `plots` docs. I don't think it should be a part of `dvc.yaml` docs - it only informs that such thing as `plots` exists.

Reply:
This is what I meant in https://github.com/iterative/dvc.org/pull/3691#discussion_r908927506. I agree with @shcheklein that we should provide some brief explanation for why there are `plots` both here and under `stages` (or the examples should make this clear) since this doc is the holistic explanation of the `dvc.yaml` structure.

> Is it primarily about bootstrapping the repo initially that doesn't have any stages?

This is actually not possible yet ☹️ and is a follow-up action. The `dvc.yaml` must already have `stages` for now.

The benefit of the global `plots` is that if your plot sources multiple outputs, there's no way to configure it in your stage outputs.